test/tests/concrete_moisture_heat_transfer/tests

```diff
@@ -602,5 +608,68 @@
       allow_test_objects = True
       detail = "Xi moisture model shall generate error if water_to_cement_ratio is set < 0.5."
     []
+    [kodur_siliceous_T_outside_range]
+      type = 'RunException'
+      input = 'concrete_thermal_moisture_properties.i'
+      expect_err = "Temperature outside of the range for the KODUR-2004 thermal model"
+      cli_args = 'Materials/concrete/thermal_model=KODUR-2004 '
+                'Materials/concrete/aggregate_type=Siliceous '
+                'Materials/concrete/moisture_model=Mensi '
+                'Materials/concrete/A=1.042e-13 '
+                'Materials/concrete/B=0.05 '
+                'Materials/concrete/cement_mass=354 '
+                'Materials/concrete/water_to_cement_ratio=0.5 '
+                'Tmax=650 '
+      allow_test_objects = True
```

Comment:
Why do we need to allow test objects? I see that parameter set for the other existing error tests, but I think it may not be needed. If not, please remove it from all the tests.

Reply:
Tmax and Tmin are the parameters used to change the temperature range in the boundary conditions so that we cover all the lines in the if-else statement for each model. They are needed as I am using these parameters in the BC's xy_data.

Reply:
I have removed it now from all test cases.